src/compiler/toitdoc_parser.cc

```diff
@@ -951,28 +955,37 @@ int ToitdocParser::peek() {
   auto text = _toitdoc_source->text();
   ASSERT(_index <= _toitdoc_source->size());
```

Comment:
Maybe also assert that the text is null terminated?
I assum that's why the index is allowed to be equal to the size?

Reply:
Yes. Added a comment and assert.